CriticalMassTests/NextRideManagerTests.swift

```diff
@@ -173,6 +173,40 @@ class NextRideManagerTests: XCTestCase {
         wait(for: [exp], timeout: 1)
     }
 
+    func testManagerShouldReturnErrorWhenRideIsCanceled() {
+        // given
+        networkLayer.mockResponse = [
+            Ride.TestData.cmBerlinDisabled
+        ]
+        let apiHandler = CMInApiHandler(networkLayer: networkLayer)
+        nextRideManager = NextRideManager(
+            apiHandler: apiHandler,
+            eventSettingsStore: RideEventSettingsStoreMock(
+                rideEventSettings: .init(
+                    isEnabled: true,
+                    typeSettings: .onlyCM,
+                    radiusSettings: .init(
+                        radius: 20,
+                        isEnabled: true
+                    )
+                )
+            )
+        )
+        // when
+        let exp = expectation(description: "Wait for response")
+        nextRideManager.getNextRide(around: CLLocationCoordinate2D.TestData.alexanderPlatz) { result in
+            switch result {
+            case .success:
+                XCTFail()
+            case .failure:
```

Comment:
We could check for the specific error `EventError.rideDisabled` here, right? I think it would add more value to the test and would be more consistent to the other existing tests. 

```suggestion
            case .failure(let error):
                XCTAssertEqual(error as! EventError, EventError.rideDisabled)
```